recipes/socat/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "socat" %}
+{% set version = "1.7.3.2" %}
+{% set md5 = "607a24c15bd2cb54e9328bfbbd3a1ae9" %}
```

Comment:
Please update this to be sha256 instead of md5. [conda-forge prefers sha256](https://github.com/conda-forge/staged-recipes/blob/master/README.md#2-how-do-i-populate-the-hash-field)